### PR TITLE
Foundations - Prerequisites: Add VM copy and paste rules & screenshot methods

### PR DESCRIPTION
**Pull request by SquidSquad484**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

#### 1.Describe the changes made:

Added:
- **Cannot copy and paste** from **guest to host OS**
  - Didn't mention one way of copy and pasting from guest to host OS because of risks and it is good practice to focus and work only in the VM
- Added **3 different ways of taking screenshots** in VM
   1. Right Ctrl + E
   2. View > Take Screenshot
   3. Whisker Menu > Screenshot
 
#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: foundations/installations/prerequisites.md [hunk 1]
```diff
@@ -111,18 +111,19 @@ While your VM is running, do the following steps:
...
-  12. In the new window that opens, right click on the white-space or any file/folder, and click **Open Terminal Here**.
-  13. In the newly opened terminal window, paste `sudo ./VBoxLinuxAdditions.run` and hit enter.
-  14. Once it finishes, close the terminal and the CD folder.
-  15. Right-click CD on the VM desktop and click **Eject Volume**. It will not eject if the CD folder is open.
-  16. Reboot your VM (which you can do by typing `reboot` and hitting enter in a terminal).
-  17. You can now maximize the VM window, create additional displays, and use many other useful features. These options are available on the VM toolbar under **View** and **Device**.
+  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. _(__note__: You cannot copy and paste to and from the guest OS and your host OS (Windows), so when copying these commands you need to open TOP in your VM through your `Web Browser` by opening the **Whisker Menu** and typing in **Web Browser**.)_
```
Comment (wise-king-sullyman) [inline foundations/installations/prerequisites.md:114]:
```suggestion
  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. _(__note__: You cannot copy and paste between your guest OS and host OS (Windows), so when copying these commands you need to open this page in your VM through your `Web Browser` by opening the **Whisker Menu** and typing in **Web Browser**.)_
```
See the above suggested edit. Additionally I think on Xubuntu `ctrl + alt + w` will open a web browser, if you're currently using Xubuntu and can confirm that I would add that in as an alternative to going through the whisker menu like in step 5.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Hey SquidSquad, thank's for doing this. I've suggested some minor changes, let me know your thoughts.

File: foundations/installations/prerequisites.md (hunk 1 above)
Comment (SquidSquad484) [reply foundations/installations/prerequisites.md]: Hi wise-king-sullyman, your edits are great; they pinpointed my exact weaknesses. I was having trouble writing that sentence and I wasn't able to find a keyboard shortcut for opening a web browser. Unfortunately, when I tried out `ctrl + alt + w` on Xubuntu nothing happened, so I suppose my instructions can remain the same.
Thanks for the help!

Comment (xandora) [reply foundations/installations/prerequisites.md]: It's listed as `Super + W`, which is probably `ctrl + shift + alt + W`.

Comment (SquidSquad484) [reply foundations/installations/prerequisites.md]: Hi xandora, unfortunately `ctrl + shift + alt + w` did not seem to do anything when I tested it out on Xubuntu, although it seemed quite plausible. Thanks for the help anyways!

Comment (xandora) [reply foundations/installations/prerequisites.md]: Oh, it's `Win + W`, at least on my Xubuntu VM.

Comment (SquidSquad484) [reply foundations/installations/prerequisites.md]: aand yep that worked

Comment (SquidSquad484) [reply foundations/installations/prerequisites.md]: Would only people with Windows install a Virtual Machine or do other OS's also need a Virtual Machine?

Comment (wise-king-sullyman) [reply foundations/installations/prerequisites.md]: I believe that we only instruct people to use a VM if they're currently on Windows.

Comment (SquidSquad484) [reply foundations/installations/prerequisites.md]: Thanks, I wasn't sure about that, I'll add `Win + W` to the instructions.

File: foundations/installations/prerequisites.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ While your VM is running, do the following steps:
   3. Install all available updates. If there are no available updates, move on to Step 5.
   4. If the **Software Updater** is stuck waiting for an **unattended upgrade** to finish, reboot the VM and start again from Step 1.
   5. Open a terminal with `ctrl + alt + t` or opening the **Whisker Menu** and typing in **Terminal** (the shortcut is obviously faster).
-  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. _(__note__: You cannot copy and paste between your guest OS and host OS (Windows), so when copying these commands you need to open this page in your VM through your `Web Browser` by opening the **Whisker Menu** and typing in **Web Browser**.)_
+  6. Copy and paste this into the terminal: `sudo apt install linux-headers-$(uname -r) build-essential dkms`. _(__note__: You cannot copy and paste between your guest OS and host OS (Windows), so when copying these commands you need to open this page in your VM through your `Web Browser` by pressing `Windows + w` (the Windows key should be between the left ctrl and alt keys).)_
```
Comment (wise-king-sullyman) [inline foundations/installations/prerequisites.md:114]: Following the pattern in number 5 above this I would mention the shortcut and the long way of using the whisker menu.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
One last detail and then it will be good to go.

**Review by wise-king-sullyman — APPROVED** [review]
Looks great, thanks for all your work on this!

**Pull request merged**